.gitignore

```diff
@@ -0,0 +1,117 @@
+.vscode
```

Comment:
This is just a standard gitignore python file, that makes sure we don't accidentally check in anything to git we don't want.